Sources/CoreAardvark/Logging/ARKLogStore.m

```diff
@@ -144,4 +165,14 @@ - (void)_applicationWillTerminate:(nullable NSNotification *)notification;
     [self waitUntilAllLogsAreConsumedAndArchiveSaved];
 }
 
+#pragma mark - Private Static Methods
+
++ (ARKDataArchive *)_dataArchiveWithPersistedLogFileURL:(nonnull NSURL *)persistedLogFileURL maximumLogMessageCount:(NSUInteger)maximumLogMessageCount;
+{
+    ARKCheckCondition(persistedLogFileURL.isFileURL, nil, @"persistedLogFileURL must be a file URL");
+    ARKCheckCondition(maximumLogMessageCount > 0, nil, @"maximumLogMessageCount must be greater than zero");
```

Comment:
I don't know if this second condition would ever be hit— since the type is `NSUInteger` I think we'd always interpret any number as positive (even if `(NSUInteger)-1` was passed in as a param)?

Reply:
A count of `0` would still hit this

Reply:
I guess 0 would be a bad way to disable the store, fair point!